config/packages/prooph_service_bus.yml

```diff
@@ -1,10 +1,6 @@
 prooph_service_bus:
   command_buses:
-    user_command_bus:
-      router:
-        type: 'prooph_service_bus.command_bus_router'
-        routes:
-            App\Application\Command\User\SignUp\SignUpCommand: App\Application\Command\User\SignUp\SignUpHandler
+    user_command_bus: ~
```

Comment:
You have already defined the command routing through a service tag in the `config/services.yaml` no need to repeat yourself (this might need no raise an error to improve DX though).